navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/NavigationService.java

```diff
@@ -79,8 +77,7 @@ public void setupNotification(Activity activity) {
     // Sets up the top bar notification
     notifyBuilder = new NotificationCompat.Builder(this)
       .setContentTitle("Mapbox Navigation")
-      .setContentText("Distance: " + routeProgress.getCurrentLegProgress().getCurrentStepProgress()
-        .getDistanceRemaining())
+      .setContentText("navigating")
```

Comment:
Planned on overhauling the notification stuff in a separate PR, The notification isn't even working right now during a navigation session...